### ISSUE TITLE
Thank you for this project :)

### ISSUE THREAD

**Issue by rachitmishra** [Genymobile/scrcpy#76]

**Comment by reflog** [comment]
Wanted to chime in as well. Amazing project. Thank you!

**Comment by swentel** [comment]
Indeed, this is amazing! My screen broke yesterday, and with this I was able to pull out all images which I hadn't synced. I can still even use my phone for playing games or answering messages, and it even helps me during my development of Android apps too.

So a big thanks to everyone who has contributed to this project!

**Comment by uoccou** [comment]
I agree completely. Have just come across this via "guiscrcpy". Using on Mandrake with a smartwatch (Kospet Prime running a version of Android7) and its made my day. Thanks to all.

**Comment by uoccou** [comment]
proof in the screencap!
![screen1](https://user-images.githubusercontent.com/417332/129589052-419dbb2c-cd0d-45b9-8024-dd308b4682fe.png)
